client/src/modules/StudentDashboard/components/NextEventCard/NextEventCard.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import CommonCard from '../CommonDashboardCard';
+import { Typography, Table } from 'antd';
+import { getAvailableTasksColumns } from './renderers';
+import { CourseScheduleItemDto } from 'api';
+
+const { Link } = Typography;
+
+type Props = {
+  nextEvents: CourseScheduleItemDto[];
+  courseAlias: string;
+};
+
+function NextEventCard({ nextEvents, courseAlias }: Props) {
+  const columns = getAvailableTasksColumns();
+
+  return (
+    <>
+      <CommonCard
+        title="Available Tasks"
+        extra={<Link href={`/course/schedule?course=${courseAlias}`}>View all</Link>}
+        content={
+          <Table
+            locale={{
+              // disable default tooltips on sortable columns
+              triggerDesc: undefined,
+              triggerAsc: undefined,
+              cancelSort: undefined,
+            }}
+            pagination={false}
+            dataSource={nextEvents}
+            rowKey="id"
+            size="middle"
+            columns={columns}
+          />
+        }
+      />
+      <style jsx>{`
+        :global(.ant-table-thead) {
+          display: none;
+        }
+      `}</style>
+    </>
+  );
+}
```

Comment:
consider the case with empty tasks, seems there shouldn't be border 
<img width="562" alt="Screenshot 2022-11-26 at 11 32 13" src="https://user-images.githubusercontent.com/1570982/204084288-fc5dd9e8-6d15-43db-aee4-e21d910f0ba3.png">

Reply:
Fixed, thank you